### PR TITLE
elasticsearch: first version

### PR DESCRIPTION
**Pull request by iksaif**
This version can only connect to elasticsearch and do
nothing else.

### CODE REVIEW

File: tools/elasticsearch/import-metrics.py [hunk 1]
```diff
@@ -85,13 +127,40 @@ def read_metrics(filename):
 
 def create(es, row):
     """Creates a metric."""
+    metric = row[0]
+    #create_metric(es, row)
```
Comment (Thib17) [inline tools/elasticsearch/import-metrics.py]: uncomment ?

File: tools/elasticsearch/import-metrics.py [hunk 2]
```diff
@@ -135,10 +204,16 @@ def main():
     )
     print('Connected:', es.info())
     if opts.cleanup:
-        es.indices.delete(index=INDEX, ignore=[400, 404])
+        es.indices.delete(index=INDEX + "metrics", ignore=[400, 404])
```
Comment (Thib17) [inline tools/elasticsearch/import-metrics.py]: /INDEX/INDEX_PREFIX/

File: biggraphite/drivers/elasticsearch.py [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+        if self.__downsampler:
+            self.__downsampler.purge()
+        if self.__delayed_writer:
+            self.__delayed_writer.write_some()
+
+    def flush(self):
+        """Flush any internal buffers."""
+        if self.__delayed_writer:
+            self.__delayed_writer.flush()
+        self.client.fush(
```
Comment (Thib17) [inline biggraphite/drivers/elasticsearch.py]: /fush/flush/

File: tools/elasticsearch/import-metrics.py (hunk 1 above)
Comment (iksaif) [reply tools/elasticsearch/import-metrics.py]: done

File: tools/elasticsearch/import-metrics.py (hunk 2 above)
Comment (iksaif) [reply tools/elasticsearch/import-metrics.py]: done

File: biggraphite/drivers/elasticsearch.py (hunk 3 above)
Comment (iksaif) [reply biggraphite/drivers/elasticsearch.py]: done

File: biggraphite/drivers/elasticsearch.py [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+            index="%s*" % self._index,
+            expand_wildcards=True,
```
[the remark this answers is not in the document]
Comment (iksaif) [reply biggraphite/drivers/elasticsearch.py]: removed

File: biggraphite/drivers/elasticsearch.py (hunk 3 above)
Comment (Thib17) [reply biggraphite/drivers/elasticsearch.py]: afaik you need client.indices.flush() according to https://elasticsearch-py.readthedocs.io/en/master/api.html#elasticsearch
'Elasticsearch' object has no attribute 'flush'

Comment (iksaif) [reply biggraphite/drivers/elasticsearch.py]: fixed

**Pull request merged**